src/map.cpp

```diff
@@ -116,7 +116,7 @@ map::map(int mapsize)
 {
     nulter = t_null;
     my_MAPSIZE = mapsize;
-    grid.resize( my_MAPSIZE * my_MAPSIZE, nullptr );
+    grid.resize( my_MAPSIZE * my_MAPSIZE * OVERMAP_LAYERS, nullptr );
```

Comment:
Please do be careful to consistently use OVERMAP_LAYERS.  Once we have something working we're probably going to want to tweak it all over the place to find a sweet spot for distance/performance.  (not doing anything wrong here, just something to keep in mind)